qiskit_ibm_runtime/_cli.py

```diff
@@ -0,0 +1,234 @@
+from .accounts.management import AccountManager, _DEFAULT_ACCOUNT_CONFIG_JSON_FILE
+from .accounts.exceptions import AccountAlreadyExistsError
+
+Channel = Literal["ibm_quantum", "ibm_cloud"]
+T = TypeVar("T")
+
+
+def entry_point() -> None:
+    """
+    This is the entry point for the `qiskit-ibm-runtime` command. At the
+    moment, we only support one script (save-account), but we want to have a
+    `qiskit-ibm-runtime` command so users can run `pipx run qiskit-ibm-runtime
+    save-account`.
+    """
+    # Use argparse to create the --help feature
+    parser = argparse.ArgumentParser(
+        prog="qiskit-ibm-runtime",
+        description="Scripts for the Qiskit IBM Runtime Python package",
+    )
+    subparsers = parser.add_subparsers(
+        title="Scripts",
+        description="This package supports the following scripts:",
+        dest="script",
+        required=True,
+    )
+    subparsers.add_parser(
+        "save-account",
+        description=(
+            "An interactive command-line interface to save your Qiskit IBM "
+            "Runtime account locally. This script is interactive-only and takes "
+            "no arguments."
+        ),
+        help="Interactive command-line interface to save your account locally.",
+    )
+    args = parser.parse_args()
+    if args.script == "save-account":
+        try:
+            SaveAccountCLI.main()
+        except KeyboardInterrupt:
+            sys.exit()
+
+
+class SaveAccountCLI:
+    """
+    This class contains the save-account command and helper functions.
+    """
+
+    @classmethod
+    def main(cls) -> None:
+        """
+        A CLI that guides users through getting their account information and
+        saving it to disk.
+        """
+        cls.print_box(["Qiskit IBM Runtime save account"])
+        channel = cls.get_channel()
+        token = cls.get_token(channel)
+        print("Verifying, this might take few seconds...")
+        try:
+            service = QiskitRuntimeService(channel=channel, token=token)
+        except (ApiException, IBMNotAuthorizedError, RequestsApiError) as err:
+            print(
+                Format.red(Format.bold("\nError while authorizing with your token\n"))
+                + Format.red(err.message)
+            )
+            sys.exit(1)
+        instance = cls.get_instance(service)
+        cls.save_to_disk(
+            {
+                "channel": channel,
+                "token": token,
+                "instance": instance,
+            }
+        )
+
+    @classmethod
+    def print_box(cls, lines: List[str]) -> None:
+        """Print lines in a box using Unicode box-drawing characters"""
+        width = max(len(line) for line in lines)
+        box_lines = [
+            "╭─" + "─" * width + "─╮",
+            *(f"│ {Format.bold(line.ljust(width))} │" for line in lines),
+            "╰─" + "─" * width + "─╯",
+        ]
+        print("\n".join(box_lines))
+
+    @classmethod
```

Comment:
These and the following methods should be `@staticmethod` since they don't use `cls`. That makes the interface more clear that you more quickly realize it's only a utility method. https://www.geeksforgeeks.org/class-method-vs-static-method-python/